api-report/sequence.api.md

```diff
@@ -52,6 +52,15 @@ import { SummarySerializer } from '@fluidframework/shared-object-base';
 import { TextSegment } from '@fluidframework/merge-tree';
 import { TypedEventEmitter } from '@fluidframework/common-utils';
 
+// @public
+export type CompressedSerializedInterval = [number, number, number, IntervalType, PropertySet];
+
+// @internal
+export function compressInterval(interval: ISerializedInterval): CompressedSerializedInterval;
```

Comment:
Do this and `compressInterval` need to be exposed? IMO it's worth adjusting the index import or putting them in a different file so that they aren't.